linux-tools/autotest-build-tools/input_file

```diff
@@ -0,0 +1,4 @@
+curl
+checkpolicy
+cracklib
+git
```

Comment:
Add all packages into input_file which has tests inside autotest and are identified for ubuntu